src/web/teap/teap/compat.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
We can get rid of this, as Python2 compatibility is not a goal.

Reply:
agree, it was by default in template, so will clean it up